builder/bundles.go

```diff
@@ -326,19 +330,20 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 			)
 			for p := range bundle.AllPackages {
 				queryString = append(queryString, p)
+				delete(bundle.AllPackages, p)
```

Comment:
Why do you need to delete the items here? Also, why delete items one by one instead of the whole list at once?

Reply:
It is only storing the name of rpm , not the full name of the rpm. I am replacing it later with full name of the rpm so initial entries are not needed. yes, it will be to better to delete it as a whole list.